matriculation-plugin/src/main/java/fi/otavanopisto/muikku/plugins/matriculation/MatriculationPermissions.java

```diff
@@ -0,0 +1,60 @@
+package fi.otavanopisto.muikku.plugins.matriculation;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import fi.otavanopisto.muikku.model.users.EnvironmentRoleArchetype;
+import fi.otavanopisto.muikku.model.workspace.WorkspaceRoleArchetype;
+import fi.otavanopisto.muikku.security.AbstractMuikkuPermissionCollection;
+import fi.otavanopisto.muikku.security.DefaultEnvironmentPermissionRoles;
+import fi.otavanopisto.muikku.security.MuikkuPermissionCollection;
+import fi.otavanopisto.muikku.security.PermissionScope;
+import fi.otavanopisto.security.Scope;
+
+@ApplicationScoped
+public class MatriculationPermissions extends AbstractMuikkuPermissionCollection implements MuikkuPermissionCollection {
+
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER, EnvironmentRoleArchetype.STUDENT } )
+  public static final String MATRICULATION_GET_EXAM = "MATRICULATION_GET_EXAM";
+
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER, EnvironmentRoleArchetype.STUDENT } )
+  public static final String MATRICULATION_GET_INITIALDATA = "MATRICULATION_GET_INITIALDATA";
+
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER, EnvironmentRoleArchetype.STUDENT } )
```

Comment:
Same as above. If enrolling is only possible for students by themselves then...

@DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.STUDENT } )

...should suffice.